### PR TITLE
[FSDE-1140] Add E2E tests to chi-vue search-input component

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**
- [x]   Added tests to chi-vue search-input component

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/1/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://d3b30632f5b1d0a8e2dd7e954a6fbe6bca6f7347-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/3/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/4/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://5ec8da4db1472b3cd5732e36dc64f8cd7b573204-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/6/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ae4e356187d95180fcd77dc1ff0670ca529814f6-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/8/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/9/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/10/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/11/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/12/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/13/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/14/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/15/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/16/. :x:

File: cypress/integration/chi-vue/search-input.spec.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+      cy.get(`${SEARCH_INPUTS_DATA_CY.DELETE}`).find(`${COMPONENT_CLASSES.CLOSE}`);
+    });
+
+    it(`should click to clean`, () => {
+      cy.get(`${SEARCH_INPUTS_DATA_CY.DELETE}`).find(`${COMPONENT_CLASSES.CLOSE}`).click();
+    });
+
+    it(`should have not value`, () => {
+      checkValue(`${SEARCH_INPUTS_DATA_CY.DELETE}`, emptyValue);
+    });
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:115]: Why are these in separate cases? These are steps of a single test case 'Clicking clean button should clean the input value' scenario

File: cypress/integration/chi-vue/search-input.spec.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+  });
+
+
+  describe('Events', () => {
+    it(`should trigger chiBlur`, () => {
+      cy.window()
+        .its(`${EXAMPLE_EVENTS.CHI_BLUR}`)
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:122]:
```suggestion
        .its(`${EVENT_EXAMPLES.BLUR}`)
```
As mentioned above, the naming needs to be more descriptive. The constant does not contain events but references of event examples.

Comment (janibolkvadze) [reply cypress/integration/chi-vue/search-input.spec.js:122]: Please,  update all the following cases as well

File: cypress/integration/chi-vue/search-input.spec.js [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+      cy.window()
+        .its(`${EXAMPLE_EVENTS.CHI_CHANGE}`)
+        .then(chiChange => {
+          const component = chiChange.$refs.testEvents;
+          const spy = cy.spy();
+          const value = 'test change';
+
+          component.$on(`${EVENTS.CHANGE}`, spy);
+          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`${ELEMENTS.INPUT}`).type(value);
+          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`${ELEMENTS.INPUT}`).focus().blur()
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:145]:
```suggestion
          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`${ELEMENTS.INPUT}`).type(value).blur();
```
Not sure if this is going to work but please try shortening the scenario like this

File: cypress/integration/chi-vue/search-input.spec.js [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+        .its(`${EXAMPLE_EVENTS.CHI_BLUR}`)
+        .then(chiBlur => {
+          const component = chiBlur.$refs.testEvents;
+          const spy = cy.spy();
+
+          component.$on(`${EVENTS.BLUR}`, spy);
+          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`${ELEMENTS.INPUT}`).focus().blur()
+            .then(() => {
+              expect(spy).to.be.called;
+            });
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:131]: Is the indentation correct here?

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

File: cypress/integration/chi-vue/search-input.spec.js [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+  DISABLED: '[data-cy="disabled-search-input-test"]',
+  ATTRS: '[data-cy="attrs-search-input-test"]',
+  DELETE: '[data-cy="delete-value-search-input-test"]',
+  EVENTS: '[data-cy="chi-events-trigger-search-input-test"]'
+};
+const ELEMENTS = {
+  INPUT: 'input'
+}
+const COMPONENT_CLASSES = {
+  CLOSE: '.-close',
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 6]
```diff
@@ -0,0 +1,216 @@
+const SEARCH_INPUTS_DATA_CY = {
+  BASE: '[data-cy="base-search-input-test"]',
+  DISABLED: '[data-cy="disabled-search-input-test"]',
+  ATTRS: '[data-cy="attrs-search-input-test"]',
+  DELETE: '[data-cy="delete-value-search-input-test"]',
+  EVENTS: '[data-cy="chi-events-trigger-search-input-test"]'
+};
+const ELEMENTS = {
+  INPUT: 'input'
+}
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+  SEARCH: '.-flat'
+}
+const EXAMPLE_EVENTS = {
+  CHI_BLUR: 'exampleChiBlur',
+  CHI_CHANGE: 'exampleChiChange',
+  CHI_CLEAN: 'exampleChiClean',
+  CHI_FOCUS: 'exampleChiFocus',
+  CHI_INPUT: 'exampleChiInput',
+  CHI_SEARCH: 'exampleChiSearch',
+}
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 8]
```diff
@@ -0,0 +1,216 @@
...
+const EVENTS = {
+  BLUR: 'chiBlur',
+  CHANGE: 'chiChange',
+  CLEAN: 'chiClean',
+  FOCUS: 'chiFocus',
+  INPUT: 'chiInput',
+  SEARCH: 'chiSearch',
+}
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 9]
```diff
@@ -0,0 +1,216 @@
...
+
+
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js:35]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 10]
```diff
@@ -0,0 +1,216 @@
...
+const checkAttr = (el, attr, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).invoke('attr', attr).should('eq', value);
+};
+
+const checkBooleanAttr = (el, attr, value) => {
+  cy.get(el).invoke('attr', attr).should('eq', value);
+};
+
+const checkValue = (el, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).should('have.value', value)
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 11]
```diff
@@ -0,0 +1,216 @@
...
+}
+
+describe('Search Input', () => {
+  before(() => {
+    cy.visit('tests/chi-vue/search-input.html');
+  });
+
+  describe('Base', () => {
+    it('should exist base search input', () => {
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 12]
```diff
@@ -0,0 +1,216 @@
...
+      cy.get(`${SEARCH_INPUTS_DATA_CY.BASE}`)
+    });
+  });
+
+  describe('Disabled', () => {
+    it('should disable search input', () => {
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 13]
```diff
@@ -0,0 +1,216 @@
...
+        .find(`${ELEMENTS.INPUT}`)
+        .should('have.attr', 'disabled');
+    });
+  });
+
+  describe('Sizes', () => {
+    const sizes = ['sm', 'md', 'lg', 'xl'];
+
+    sizes.forEach((size) => {
+      it(`should have -${size} size`, () => {
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/17/. :x:

File: cypress/integration/chi-vue/search-input.spec.js [hunk 14]
```diff
@@ -0,0 +1,216 @@
...
+  SEARCH: 'chiSearch',
+}
+
+
+const checkSizeClass = (el, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).should('have.class', value);
+};
+
+const checkAttr = (el, attr, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).invoke('attr', attr).should('eq', value);
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 15]
```diff
@@ -0,0 +1,216 @@
...
+        checkSizeClass(`[data-cy='${size}-search-input-test']`, `-${size}`);
+      });
+    });
+  });
+
+  describe('Attrs', () => {
+    const attrs = ['name', 'placeholder'];
+    const booleanAttrs = ['portal', 'prevent-value-mutation']
+    const booleanValue = 'true';
+    const value = 'test';
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 16]
```diff
@@ -0,0 +1,216 @@
...
+
+    attrs.forEach((attr) => {
+      it(`should have ${attr} attr`, () => {
+        checkAttr(`${SEARCH_INPUTS_DATA_CY.ATTRS}`, attr, value);
+      });
+    });
+
+    booleanAttrs.forEach((attr) => {
+      it(`should have ${attr} attr at true`, () => {
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 17]
```diff
@@ -0,0 +1,216 @@
...
+        checkBooleanAttr(`${SEARCH_INPUTS_DATA_CY.ATTRS}`, attr, booleanValue);
+      });
+    });
+  });
+
+  describe('Check delete value', () => {
+    const value = 'test';
+    const emptyValue = '';
+
```
[the remark this answers is not in the document]
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js (hunk 1 above)
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Deleted.

File: cypress/integration/chi-vue/search-input.spec.js (hunk 2 above)
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js (hunk 3 above)
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: It works! Solved.

File: cypress/integration/chi-vue/search-input.spec.js (hunk 4 above)
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js:123]: Yes it is (spaces 2), solved.

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://96f232ce06ae92d35e8f2429d2e798036f1ed52c-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/19/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/20/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://7f1b68d369872cbbb81f97d77a2b2f094c9d7c34-chi-pr.enterprise.ctl.io

File: cypress/integration/chi-vue/search-input.spec.js [hunk 18]
```diff
@@ -0,0 +1,207 @@
...
+  FOCUS: 'chiFocus',
+  INPUT: 'chiInput',
+  SEARCH: 'chiSearch',
+};
+
+const checkSizeClass = (el, value) => {
+  cy.get(el).find(`${ELEMENTS.INPUT}`).should('have.class', value);
+};
+
+const checkAttr = (el, sel, attr, value) => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:40]: The element should be the one you check the attribute for. What you are doing here is receive 1) A parent element; 2) A selector by which you find the target element. I suggest you receive the target element directly from outside.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 19]
```diff
@@ -0,0 +1,207 @@
...
+    sizes.forEach((size) => {
+      it(`Should have the class -${size} size`, () => {
+        checkSizeClass(`[data-cy='${size}-search-input-test']`, `-${size}`);
+      });
+    });
+  });
+
+  describe('Attrs', () => {
+    const attrs = ['name', 'placeholder'];
+    const booleanAttrs = ['portal', 'prevent-value-mutation']
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:91]:
```suggestion
    const booleanAttrs = ['portal']
```

Prevent-value-mutation is not a Chi Vue Search Input prop and should be removed.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 20]
```diff
@@ -0,0 +1,207 @@
...
+
+    attrs.forEach((attr) => {
+      it(`Should have ${attr} attr`, () => {
+        checkAttr(`${SEARCH_INPUTS_DATA_CY.ATTRS}`, `${ELEMENTS.INPUT}`, attr, 'test');
+      });
+    });
+
+    booleanAttrs.forEach((attr) => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:99]: This iteration is not necessary as prevent-value-mutation element should be removed from the booleanAttrs array.

File: cypress/integration/chi-vue/search-input.spec.js [hunk 21]
```diff
@@ -0,0 +1,207 @@
+const SEARCH_INPUTS_DATA_CY = {
+  BASE: '[data-cy="base-search-input-test"]',
+  DISABLED: '[data-cy="disabled-search-input-test"]',
+  ATTRS: '[data-cy="attrs-search-input-test"]',
+  DELETE: '[data-cy="delete-value-search-input-test"]',
+  EVENTS: '[data-cy="chi-events-trigger-search-input-test"]'
+};
+const ELEMENTS = {
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:8]: Why is this necessary?

File: cypress/integration/chi-vue/search-input.spec.js [hunk 22]
```diff
@@ -0,0 +1,207 @@
...
+          const component = chiChange.$refs.testEvents;
+          const spy = cy.spy();
+          const value = 'test change';
+
+          component.$on(`${EVENTS.CHANGE}`, spy);
+          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`${ELEMENTS.INPUT}`).type(value).blur()
+            .then(() => {
+              checkValue(`${SEARCH_INPUTS_DATA_CY.EVENTS}`, value);
+              expect(spy).to.be.called;
+            });
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:140]: The indentation seems a bit off. Is it indented using 2 spaces?

File: cypress/integration/chi-vue/search-input.spec.js [hunk 23]
```diff
@@ -0,0 +1,207 @@
...
+  DELETE: '[data-cy="delete-value-search-input-test"]',
+  EVENTS: '[data-cy="chi-events-trigger-search-input-test"]'
+};
+const ELEMENTS = {
+  BUTTON: 'button',
+  DIV: 'div',
+  ICON: 'i',
+  INPUT: 'input'
+};
+const COMPONENT_CLASSES = {
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:14]: Are we checking `chi-form__item`, `chi-search__input`classes, `-icon--right` classes?

File: cypress/integration/chi-vue/search-input.spec.js [hunk 24]
```diff
@@ -0,0 +1,207 @@
...
+        .then(chiClean => {
+          const component = chiClean.$refs.testEvents;
+          const spy = cy.spy();
+
+          component.$on(`${EVENTS.CLEAN}`, spy);
+          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`${ELEMENTS.INPUT}`).type('test')
+          cy.get(`${SEARCH_INPUTS_DATA_CY.EVENTS}`).find(`.${COMPONENT_CLASSES.CLOSE}`).click()
+            .then(() => {
+              expect(spy).to.be.called;
+            });
```
Comment (janibolkvadze) [inline cypress/integration/chi-vue/search-input.spec.js:156]: Here we are checking if the event has been emitted, but we are not checking if the value has been set to an empty array (basically, the clean functionality).

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

File: cypress/integration/chi-vue/search-input.spec.js (hunk 19 above)
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js (hunk 20 above)
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Solved.

File: cypress/integration/chi-vue/search-input.spec.js (hunk 21 above)
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js]: Removed and solved.

File: cypress/integration/chi-vue/search-input.spec.js (hunk 22 above)
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js:142]: Yes it is, i'm using  2 spaces.

File: cypress/integration/chi-vue/search-input.spec.js (hunk 23 above)
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js:8]: Now yes, solved.

File: cypress/integration/chi-vue/search-input.spec.js (hunk 24 above)
Comment (josecarlosgonzalezv) [reply cypress/integration/chi-vue/search-input.spec.js:159]: Solved.

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/21/. :x:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f7cf5e7c96f073a239598435bae3b44152628ee3-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/23/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/24/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/25/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/26/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/27/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://1e9d323b88cf8690f1c0fa36d80950299db9f745-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/28/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://8610b4e7107e93f5ae6bd35671151ed099db260c-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/29/. :x:

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/30/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/31/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://4a9484b10e2ef61cab49328f87a34660e851a8b3-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-105/32/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://8efa1c187f3ace45dcfe54f40244f139464563b9-chi-pr.enterprise.ctl.io

**Review by janibolkvadze — APPROVED** [review]
